cli/src/main/kotlin/Main.kt

```diff
@@ -141,4 +165,25 @@ object Main : CommandWithHelp() {
 
         println()
     }
+
+    private fun loadConfig(): OrtConfiguration {
+        val argConfig = ConfigFactory.parseMap(configArguments, "Command line").withOnlyPath("ort")
```

Comment:
Nit: Maybe "argsConfig" (plural) here, as that abbreviation seems to be a bit more common?

Reply:
Done.